### ISSUE TITLE
Horizontal scroll in style guide appears on every page that has vertical scrolling

### ISSUE THREAD

**Issue by shaal** [fourkitchens/emulsify#261]
You can see that behavior here -
https://fourkitchens.github.io/emulsify/pattern-lab/public/?p=all
and also on any other page in the style guide that has vertical scrolling.

**Comment by ModulesUnraveled** [comment]
What browser, OS, and versions are you using? I don't see a horizontal scrollbar when I load that page.

**Comment by shaal** [comment]
I tested it on -
Ubuntu 18.10 + Chromium Version 66.0.3359.181
Ubuntu 18.10 + Chrome Version 67.0.3396.87
Ubuntu 18.10 + Firefox Nightly 61.0a1 (2018-05-03)
and even tried it on windows using https://www.browserling.com/
![emulsify-styleguide-on-chrome--with-horizontal-scroll-bar](https://user-images.githubusercontent.com/22901/41873535-3f324b98-7893-11e8-8bd5-ff7f97ac2f67.png)

From what I see, the cause is the margin rule here -
```
.pl .sg-subtype h2, .sg-main .sg-subtype h2 {
    font-size: 3rem;
    text-transform: uppercase;
    background-color: #333;
    margin: 0 calc(-50vw + 50%) 1rem calc(-50vw + 50%);
}
```

**Comment by ModulesUnraveled** [comment]
It looks like it's happening on browsers that always show the vertical scrollbar.

If you were on a mac (or something else?) that has a "only show the scrollbars when in use" setting, I suspect you wouldn't see the horizontal scrollbar.

But since yours is always showing, it looks like it's affecting the calculated viewport width, and throwing off the margin...

Different browsers (and OSs for that matter) make things difficult... I think we should all start using the same hardware and software. Then we won't have issues like this. 😄 

Seriously though... I'm not sure what the right answer is here. I'll have to think on it.

**Comment by ModulesUnraveled** [comment]
**UPDATE** Yeah, I just went and toggled my system setting to "Always show scrollbars" and could reproduce the horizontal scrolling issue. Bummer... I guess we all have to use the same software settings too. 😆 

More thinking required

**Comment by shaal** [comment]
resolved by #269
